library_generation/test/utils/generation_config_comparator_unit_tests.py

```diff
@@ -58,6 +61,24 @@ def test_compare_config_not_change(self):
         )
         self.assertTrue(len(result.change_to_libraries) == 0)
 
+    def test_compare_non_monorepo_configs__creates_default_change(self):
```

Comment:
What happens if googleapis commit changed in a non-monorepo config? Does the description.txt get generated?

Reply:
It was a great test to add, but I changed the approach following @blakeli0's suggestion. PR description generation won't be modified anymore.